workspace/notebook/py_sb_horizon_harmonised_nsip_project.json

```diff
@@ -781,7 +783,7 @@
 					"                    ,IFNULL(CAST(applicantwebaddress AS String), '.')\n",
 					"                    ,IFNULL(CAST(applicantphonenumber AS String), '.')\n",
 					"                    ,IFNULL(CAST(applicantdescriptionofproject AS String), '.')\n",
-					"                    ,IFNULL(CAST(caseNodeId AS String), '.')\n",
+					"                   --,IFNULL(CAST(HorizonCaseNumber AS int), '.')\n",
```

Comment:
Should this be commented? just curious

Reply:
if you mean line 786, no it shouldnt be commented

Reply:
done now :)